mesonbuild/interpreter/interpreterobjects.py

```diff
@@ -139,7 +139,7 @@ def require_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> coredat
         assert isinstance(error_message, str)
         if self.value == 'enabled':
             prefix = f'Feature {self.held_object.name} cannot be enabled'
-            prefix = prefix + ': ' if error_message else ''
+            prefix += ': ' if error_message else ''
```

Comment:
```python
if error_message:
    prefix += ': '
```

maybe?

This code looks a bit confusing, and it seems like the first person here thought that
```python
             prefix = prefix + ': ' if error_message else ''
```

was actually 

```python
             prefix = prefix + (': ' if error_message else '')
```

I'm wondering what we can do to make it more obvious. :)

Reply:
add the parens? :D

Reply:
Amended to use:
```python
if error_message:
    prefix += ': '
```